content/release-10-16-0/devices-10-16-0-bundle/10-16-0-256.md

```diff
@@ -57,4 +57,10 @@ Issue</th>
 <td>DM-2037</td>
 </tr>
 
+<tr>
+<td>OPC UA</td>
+<td>If the namespace is not defined for a specific node, a missing error handler in the past prevented completion of the scan operation. This is now fixed by skipping nodes that cannot be read. The opcua-device-gateway logs include further details in such cases. The log level INFO contains the failure message and the log level DEBUG contains the stacktrace.</td>
```

Comment:
this one is no longer up for review because we edited it earlier, correct?

Reply:
You have already reviewed it once :-)

Reply:
alrighty :)